plugins/experimental/memcache/tsmemcache.cc

```diff
@@ -388,11 +388,13 @@ MC::write_binary_response(const void *d, int hlen, int keylen, int dlen)
       case VC_EVENT_EOS:                             \
         if ((VIO *)data == rvio)                     \
           break;                                     \
+      /* fallthrough */                              \
       case VC_EVENT_READ_READY:                      \
         return EVENT_CONT;                           \
       case VC_EVENT_WRITE_READY:                     \
         if (wvio->buffer.reader()->read_avail() > 0) \
           return EVENT_CONT;                         \
+      /* fallthrough */                              \
```

Comment:
Hmmm, I wonder if instead of doing the // fallthrough here, we could just make each case be explicit? I mean, the savings of the fall through is very marginal, we could just copy up the explicit return in the else cases etc.?